include/mbedtls/md.h

```diff
@@ -36,7 +36,8 @@
 
 /*
  * - MBEDTLS_MD_CAN_xxx is defined if the md module can perform xxx.
- * - MBEDTLS_MD_xxx_VIA_PSA is defined if the md module performs xxx via PSA.
+ * - MBEDTLS_MD_xxx_VIA_PSA is defined if the md module performs xxx via PSA
+ *   (when PSA Crypto is initialized).
```

Comment:
Minor: this applies to all lines below, or maybe it would be just better to say that "PSA can perform xxx", just as with MD above.